js/lib/components/feedback/selection.tsx

```diff
@@ -0,0 +1,98 @@
+import { VirtualElement } from "@popperjs/core";
+import React, { useCallback, useEffect, useState } from "react";
+import Highlighter from "web-highlighter";
+
+import { HIGHLIGHT_STORAGE_KEY } from "../../entryPoints/feedback";
+import FeedbackModal from "./modal";
+import SelectionTooltip from "./tooltip";
+
+type SelectionRendererProps = { highlighter: Highlighter; stored: any[] | undefined };
+let SelectionRenderer: React.FC<SelectionRendererProps> = ({ highlighter, stored }) => {
+  // id of feedback highlight currently being hovered over
+  const [hovered, setHovered] = useState<false | string>(false);
```

Comment:
Just a style thing, but I would use `null` instead of `false` to represent `None`.